CONTRIBUTING.md

```diff
@@ -19,3 +19,20 @@ If you are signing as an individual, we recommend that you talk to your employer
 If your employer has rights to intellectual property that you create, such as your contributions, you represent that you have received permission to make contributions on behalf of that employer, that your employer has waived such rights for your contributions, or that your employer has executed a separate CLA with F5.
 
 If you are signing on behalf of a company, you represent that you are legally entitled to grant the license recited therein. You represent further that each employee of the entity that submits contributions is authorized to submit such contributions on behalf of the entity pursuant to the CLA.
+
+## Release process
+* Merge changes into `master` branch
+* Update CHANGELOG with date for release (likely today)
+* `git checkout master`
+* `git pull`
+* `npm version`
+  * Note: depending on what version is being released this command might be changed to `npm version patch` or another command
+* `git push`
+* `git push origin v{newVersion}`
+  * Example: `git push origin v1.3.2`
+  * Note: This will push the new tags created by the previous `npm version` command. You can confirm this via the GUI.
+* Via the GUI create a new release branch, using the tag just pushed. Copy the relevant CHANGELOG entry into the release version message.
```

Comment:
Same as above with respect to "GUI".

Also, we are creating a "GitHub release" and not a "release branch"